projects/core/src/configurator/commons/store/selectors/configurator-ui.selector.ts

```diff
@@ -46,9 +46,11 @@ export const isGroupVisited = (
   ownerKey: string,
   groupId: string
 ): MemoizedSelector<StateWithConfiguration, Boolean> => {
-  return createSelector(getUiStateForOwner(ownerKey), (details) =>
-    StateUtils.entitySelector(details.groupsVisited, groupId)
-  );
+  return createSelector(getUiStateForOwner(ownerKey), (details) => {
+    if (details) {
```

Comment:
We should have the condition covered by a karma test

Reply:
Test added